### PR TITLE
Desktop Version

### PR DESCRIPTION
**Pull request by sunga12**
- Introduction Page Done
- Recent Works Page Done

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -79,9 +94,26 @@ <h2 class="professional">Professional Art Printing Data</h2>
...
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">Ruby</a></li>
+          </ul>
+  
+        </div>
+        <button a href="#" class="project-link-btn">See Project</a></button>
+        
+      </div>
+
+      <div class="project project3">
```
Comment (aamirkhan2478) [inline index.html:114]: - [ ] Kindly stick to the Figma design your design is not the same as the Figma design your overall design is very good :clap: but there are a few changes check below the comparison between your and Figma design :+1: 

**Your Design**
Screenshot from 2023-03-30 17-58-26

**Figma Design**
Screenshot from 2023-03-29 23-58-57

- [x] If you check your cards `height` is not the same as Figma design's cards 'height' so kindly fix the `height` of all cards.
- [x] If you check the first `card` is changed to the other `cards` you can see there is one `button` and also a background `overlay` is hidden and the `picture` is shown clearly so please add this functionality whenever someone hovers over any card the overlay will be hidden and the button will be shown :+1: check below the video :point_down: and forget about the design in the video just check when I hover over any of card it is changing the state.

[Video](https://user-images.githubusercontent.com/45226848/228850385-8c6b9d1c-92bd-43a0-a91c-7e1b922f14c9.webm)

**_Note: These changes will be implemented only in desktop version not in mobile version_**

File: index.html [hunk 2]
```diff
@@ -11,39 +11,54 @@
...
+    </ul>
 </section>
 
 <section class="Recent-works">
 
+  <div class="recent-works-heading">
     <h1 class="recent-works-title">My Recent Works</h1>
-    <hr/>
+  
+    <div class="line"></div>
```
Comment (aamirkhan2478) [inline index.html:54]: - [ ] Kindly Stick to the Figma design as I mentioned above your design is not like the Figma design kindly fix it :+1: 

**Your design**
Screenshot from 2023-03-30 18-02-08

**Figma design**
Screenshot from 2023-03-30 18-02-55

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @sunga12 , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good Effort
✔️ Your design is good.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @sunga12 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**